bok_choy/browser.py

```diff
@@ -204,8 +205,13 @@ def browser_check_func():
             LOGGER.debug('Failed to instantiate browser: ' + err.strerror)
             return False, None
 
+        except WebDriverException as err:
+            msg = str(err)
```

Comment:
What's the difference between str(err) and err.strerror (which is used in the other except block above)? Should we standardize on one or the other?